modules/src/food.py

```diff
@@ -0,0 +1,93 @@
+import os
+import json
+import requests
+import random
+import difflib
+
+import config
+from templates.generic import *
+from templates.text import TextTemplate
+
+ZOMATO_API_KEY = os.environ.get('ZOMATO_API_KEY', config.ZOMATO_API_KEY)
+
+def process(input,entities):
+    output = {}
+    try:
+        requested_location = entities['restaurant_location'][0]['value']
+
+        header = {'user-key': ZOMATO_API_KEY, 'Accept': 'application/json'}
+        query = requests.get('https://developers.zomato.com/api/v2.1/locations?query=' + requested_location, headers=header).json()
```

Comment:
Are we sure that JSON conversion wouldn't fail? What happens when the connection is dropped or malformed response received. Good to have it handled :)

Reply:
lets start with renaming the query as the response that is more readable ,and then we shall do ‘‘‘ if response.ok'''

Reply:
lets start with renaming the query as the response that is more readable , and then we shall do 
```
if response.ok:
    content = response.json()
    try:
        location=content['location_suggestions'][0]
    except (KeyError,IndexError):
        #do what you gotta do.
    #do normal thing
```
This is still hacky way but at least we have something to catch if fails.An I hope this will make it clear :)

Reply:
updated with your changes, is that what you were looking for?